src/app/analyze/stack/stack-details/stack-details.component.html

```diff
@@ -3,166 +3,203 @@
       <span id="packageInfo">
-        <h2> meanjs 0.5.0</h2>
+        <h2> {{ packageName }} - {{ packageVersion}}</h2>
       </span>
     </div>
 
   </div>
 
 
-  <div id="result-body" class="container widget-row widget-span" style="display:block">
+  <div id="result-body" class="container widget-row widget-span" style='display:block'>
     <!-- <div class="widget-span"> -->
     <!-- <br> -->
     <!-- <div class="stack_info_container"> -->
 
     <div class="row">
       <div class="col-sm-6 col-lg-6 render-stack-tiles ">
-        <h4><span class="nav-text">
-            <span class="ui-icon ui-icon-info"></span>Usage</span>
-          <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#usage-stack">
-            <!-- <img title='Overview on how this stack is being used. Click for more info.'
-             src="../assets/icons/266-question.svg"></img> -->
-          </a>
-        </h4>
-        <div id="usage">
-          <p> Average Public Component Usage: {{averageUsage}}</p>
-          <p> Components with Low Public Usage: {{lowPublicUsageComponents}}</p>
-          <p> Components Redistributed by Red Hat: {{redhatDistributedComponents}}</p>
+        <div class="card-pf card-pf-accented card-pf card-pf-view">
+          <h2 class="card-pf-title">
+            <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#usage-stack"><span class="pf pficon-help"></span><span class="card-pf-view-count"></span> Usage</a>
+          </h2>
+          <div class="card-pf-body">
+            <div id="usage">
+              <p> Average Public Component Usage: {{averageUsage}}</p>
+              <p> Components with Low Public Usage: {{lowPublicUsageComponents}}</p>
+              <p> Components Redistributed by Red Hat: {{redhatDistributedComponents}}</p>
+            </div>
+            <p class="card-pf-view-notifications">
+            </p>
+          </div>
         </div>
       </div>
       <div class="col-sm-6 col-lg-6 render-stack-tiles ">
-        <h4>
-          <span class="nav-text">Popularity</span>
-          <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#popularity-stack">
-            <!-- <img title='Overview on how popular this stack is. Click for more info.'
-              src="icons/266-question.svg"></img> -->
-          </a>
-        </h4>
-        <div id="popularity">
-          <p> Average Component GitHub Stars: {{averageStars}}</p>
-          <p> Average Component GitHub Forks: {{averageForks}}</p>
-          <p> Components with Low Popularity: {{lowPopularityComponents}}</p>
-
+        <div class="card-pf card-pf-accented card-pf-view">
+          <h2 class="card-pf-title">
+            <a href="https://gitlab.cee.redhat.com/bayesian/Bayesian/tree/master/docs/stack-analysis.md#popularity-stack"><span class="pf pficon-help"></span><span class="card-pf-view-count"></span> Popularity</a>
```

Comment:
Why do you have reference to https://gitlab.cee.redhat.com here. This is not a public URL. It does not work on this publicly running application.